other/litefs.yml

```diff
@@ -21,12 +21,16 @@ proxy:
 # You must specify "experiement.enable_consul" for FLY_CONSUL_URL to be available.
 lease:
   type: 'consul'
-  candidate: true
-  # If you'd like to control the candidate's possible region, you can use some
-  # simple expressions for this value, like so:
-  # candidate: ${FLY_REGION == 'sjc'}
+  candidate: ${FLY_REGION == 'sjc'}
```

Comment:
For new apps on Fly.io, we have a `PRIMARY_REGION` environment variable available so you can do:

```
candidate: ${FLY_REGION == PRIMARY_REGION}
```